Magefile.go

```diff
@@ -19,11 +19,12 @@ func DataGenerate() error {
 
 // Protobuf generates protobuf files.
 func Protobuf() error {
-	if err := sh.RunV("./scripts/protobuf-check.sh"); err != nil {
-		return err
-	}
+	return sh.RunV("buf", "generate", "proto", "--template", "./proto/buf.gen.yaml")
+}
 
-	return sh.RunV("./proto/generate.sh")
+// ProtobufValidate validate breaking changes in protobuf files.
+func ProtobufValidate() error {
```

Comment:
Nit: Maybe define `Protobuf` as a mage namespace and have:

```go
type Protobuf mg.Namespace

func (Protobuf) Generate()
func (Protobuf) Validate()
```

so they can be run like this instead:

```
mage protobuf:generate
mage protobuf:validate
```

WDYT?

Reply:
Good idea 👍